sources/Session.php

```diff
@@ -57,7 +57,10 @@ function loadSession()
 		// This is here to stop people from using bad junky PHPSESSIDs.
 		if (isset($_REQUEST[session_name()]) && preg_match('~^[A-Za-z0-9,-]{16,64}$~', $_REQUEST[session_name()]) == 0 && !isset($_COOKIE[session_name()]))
 		{
-			$session_id = md5(md5('elk_sess_' . time()) . mt_rand());
+			require_once(SUBSDIR . '/TokenHash.class..php');
```

Comment:
LOL ... I'll get that fixed ... I'll try it with the autoloader, I know it missed from time to time in subs files (and I should have fixed that but did not).  I'll give it a try and see if its fatal !